zeroex/asset_data_decoder.go

```diff
@@ -55,10 +68,21 @@ type ERC1155AssetData struct {
 type ERC20BridgeAssetData struct {
 	TokenAddress  common.Address
 	BridgeAddress common.Address
-	Calldata      []byte
+	BridgeData    []byte
+}
+
+// StaticCallAssetData represents a StaticCallAssetData
+type StaticCallAssetData struct {
+	StaticCallTargetAddress common.Address
+	StaticCallData          []byte
```

Comment:
Are there somehow the canonical field names? Otherwise, I'd be in favor of dropping the `StaticCall` prefix as it stutters.

Reply:
These are the canonical field names from the contracts. I'd prefer to stick with those in these structures and in the ABIs so that the contracts can be the single source of truth for these names.